pkg/minikube/bootstrapper/images/kubeadm_test.go

```diff
@@ -75,43 +75,43 @@ func TestKubeadmImages(t *testing.T) {
 			"mirror.k8s.io/k8s-minikube/storage-provisioner:" + version.GetStorageProvisionerVersion(),
 		}},
 		{"v1.24.0", "", false, []string{
-			"k8s.gcr.io/kube-proxy:v1.24.0",
-			"k8s.gcr.io/kube-scheduler:v1.24.0",
-			"k8s.gcr.io/kube-controller-manager:v1.24.0",
-			"k8s.gcr.io/kube-apiserver:v1.24.0",
-			"k8s.gcr.io/coredns/coredns:v1.8.6",
-			"k8s.gcr.io/etcd:3.5.3-0",
-			"k8s.gcr.io/pause:3.7",
+			"registry.k8s.io/kube-proxy:v1.24.0",
+			"registry.k8s.io/kube-scheduler:v1.24.0",
+			"registry.k8s.io/kube-controller-manager:v1.24.0",
+			"registry.k8s.io/kube-apiserver:v1.24.0",
+			"registry.k8s.io/coredns/coredns:v1.8.6",
+			"registry.k8s.io/etcd:3.5.3-0",
+			"registry.k8s.io/pause:3.7",
 			"gcr.io/k8s-minikube/storage-provisioner:" + version.GetStorageProvisionerVersion(),
 		}},
 		{"v1.23.0", "", false, []string{
-			"k8s.gcr.io/kube-proxy:v1.23.0",
-			"k8s.gcr.io/kube-scheduler:v1.23.0",
-			"k8s.gcr.io/kube-controller-manager:v1.23.0",
-			"k8s.gcr.io/kube-apiserver:v1.23.0",
-			"k8s.gcr.io/coredns/coredns:v1.8.6",
-			"k8s.gcr.io/etcd:3.5.1-0",
-			"k8s.gcr.io/pause:3.6",
+			"registry.k8s.io/kube-proxy:v1.23.0",
+			"registry.k8s.io/kube-scheduler:v1.23.0",
+			"registry.k8s.io/kube-controller-manager:v1.23.0",
+			"registry.k8s.io/kube-apiserver:v1.23.0",
+			"registry.k8s.io/coredns/coredns:v1.8.6",
+			"registry.k8s.io/etcd:3.5.1-0",
+			"registry.k8s.io/pause:3.6",
 			"gcr.io/k8s-minikube/storage-provisioner:" + version.GetStorageProvisionerVersion(),
 		}},
 		{"v1.22.0", "", false, []string{
-			"k8s.gcr.io/kube-proxy:v1.22.0",
-			"k8s.gcr.io/kube-scheduler:v1.22.0",
-			"k8s.gcr.io/kube-controller-manager:v1.22.0",
-			"k8s.gcr.io/kube-apiserver:v1.22.0",
-			"k8s.gcr.io/coredns/coredns:v1.8.4",
-			"k8s.gcr.io/etcd:3.5.0-0",
-			"k8s.gcr.io/pause:3.5",
+			"registry.k8s.io/kube-proxy:v1.22.0",
+			"registry.k8s.io/kube-scheduler:v1.22.0",
+			"registry.k8s.io/kube-controller-manager:v1.22.0",
+			"registry.k8s.io/kube-apiserver:v1.22.0",
+			"registry.k8s.io/coredns/coredns:v1.8.4",
+			"registry.k8s.io/etcd:3.5.0-0",
+			"registry.k8s.io/pause:3.5",
 			"gcr.io/k8s-minikube/storage-provisioner:" + version.GetStorageProvisionerVersion(),
 		}},
 		{"v1.16.0", "", false, []string{
-			"k8s.gcr.io/kube-proxy:v1.16.0",
-			"k8s.gcr.io/kube-scheduler:v1.16.0",
-			"k8s.gcr.io/kube-controller-manager:v1.16.0",
-			"k8s.gcr.io/kube-apiserver:v1.16.0",
-			"k8s.gcr.io/coredns:1.6.2",
-			"k8s.gcr.io/etcd:3.3.15-0",
-			"k8s.gcr.io/pause:3.1",
+			"registry.k8s.io/kube-proxy:v1.16.0",
```

Comment:
not changing anything for 1.16 ?

Reply:
I must have accidentally just replaced that 1 line vs all in the file, will update